applications/band-grants.md

```diff
@@ -0,0 +1,164 @@
+- An indication of why your team is interested in creating this project.
+    - Astar has a robust foundation and ecosystem, making it an attractive platform for many talented builders to develop on in the near future. Our team has been integrated with Astar EVM for over a year now, and we’ve already requested our technical team review the technical specifications and documentation of Astar Wasm. Our team has significant experience integrating with EVM, WASM, and Rust-based projects, which gives us the expertise needed to provide excellent services for dapps looking to build on Astar Wasm. By leveraging our technical know-how, we are confident that we can provide strong support to developers building on the Astar Wasm platform.
+
+### Ink! Ecosystem Impact
+
+Please clearly describe how exactly your project will benefit the ecosystem. If it's infrastructure - how it's going to be applied and by who. If it's canary dapp we would you to clearly demonstrate how this code will be used by other people. If it's technical showcase we would like you to demonstrate how ink! will be promoted using your results. 
+
+If it's a business case we would like to see business model and future production plans. 
+
+- Band Protocol offers a wide range of real-world data services, including cryptocurrency prices, foreign exchange rates, and commodity prices. Our services are currently available on 19 chains, including Ethereum, BNB Chain, Avalanche, and Cronos. We have also integrated with various projects on their testnets to ensure seamless data delivery. We have collaborated with lending, perpetual, yield protocols and more to provide accurate and reliable data that supports a variety of use cases. By partnering with Band Protocol, Dapps can access the data they need to build innovative and secure blockchain applications that meet the needs of their users.
+
+### Project Details
+
+We expect the teams to already have a solid idea about the project expected final state. Therefore, we ask the teams to submit (where relevant):
+
+- Data models / API specifications of the core functionality
+- An overview of the technology stack to be used
+- Documentation of core components, protocols, architecture, etc. to be deployed
+- PoC/MVP or other relevant prior work or research on the topic
+- What your project is _not_ or will _not_ provide or implement
+  - This is a place for you to manage expectations and to clarify any limitations that might not be obvious
+- Do you need an audit for the contacts? **YES/NO**
+(don’t add it as part of Milestones! The auditor will be chosen among Auditor track participants)
+- CATEGORY: Infrastructure OR Canary Dapp OR Technical Showcase
```

Comment:
Could you elaborate on the questions above?

Reply:
Provided the missing info on the updated doc.